src/algoliasearch.helper.js

```diff
@@ -1238,10 +1238,12 @@ AlgoliaSearchHelper.prototype._hasDisjunctiveRefinements = function(facet) {
     this.state.disjunctiveRefinements[facet].length > 0;
 };
 
-AlgoliaSearchHelper.prototype._change = function(newState) {
+AlgoliaSearchHelper.prototype._change = function(newState, resetPage) {
+  var isPageReset = resetPage || false;
+
   if (newState !== this.state) {
     this.state = newState;
-    this.emit('change', this.state, this.lastResults);
+    this.emit('change', this.state, this.lastResults, isPageReset);
```

Comment:
I'm not too much a fan of this boolean trap, what if we make it an options object? not sure if there's more things possible, and how long we plan on keeping the emitters though 🤔

Reply:
Yes indeed, we could use an object. We already have BCs anyway we can move the arguments to an object. It would help to understand which value is provided to the listener.

We have to update the others too, it makes no sense to keep both at the same time. I'll only update the shape of the `change` event in the PR. I'll do the others (`search`, `result`, ...) in another one afterward.

Reply:
Resolved in https://github.com/algolia/algoliasearch-helper-js/pull/673/commits/5698b7939130b2a21fa4e20167c9c279e77440c8